packages/sqlite3/sqlite3.4.1.1/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "1.2"
+maintainer: "Markus Mottl <markus.mottl@gmail.com>"
+authors: [ "Markus Mottl <markus.mottl@gmail.com>"
+           "Christian Szegedy <csdontspam@metamatix.com>" ]
+license: "Expat"
+homepage: "http://mmottl.github.io/sqlite3-ocaml"
+dev-repo: "https://github.com/mmottl/sqlite3-ocaml.git"
+bug-reports: "https://github.com/mmottl/sqlite3-ocaml/issues"
+tags: [ "clib:sqlite3" "clib:pthread"  ]
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"] { os != "darwin" }
+  ["env" "SQLITE3_OCAML_BREWCHECK=1"
+   "ocaml" "setup.ml" "-build"] { os = "darwin" }
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
+  ["ocamlfind" "remove" "sqlite3"]
+]
+build-test: [
+  ["ocaml" "setup.ml" "-configure" "--enable-tests"]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-test"]
+]
+build-doc: [ "ocaml" "setup.ml" "-doc" ]
+depends: [
+  "ocamlbuild" {build}
+  "ocamlfind" {build & >= "1.3.1"}
+  # Included from _opam file
+  "conf-pkg-config" {build}
+]
+available: [ ocaml-version >= "3.12" ]
+depexts: [
+  [["debian"] ["libsqlite3-dev"]]
+  [["freebsd"] ["database/sqlite3"]]
+  [["openbsd"] ["database/sqlite3"]]
+  [["ubuntu"] ["libsqlite3-dev"]]
+  [["centos"] ["sqlite-devel"]]
+  [["rhel"] ["sqlite-devel"]]
+  [["fedora"] ["sqlite-devel"]]
+  [["alpine"] ["sqlite-dev"]]
+  [["osx" "homebrew"] ["sqlite3"]]
+  [["osx" "macports"] ["sqlite3"]]
```

Comment:
any chance you could add `[["opensuse" ["sqlite3-devel"]]` to the depexts list? The CI now tests that too and it'll increase your support coverage. Not a blocker to merge though.